object:
  object_name: "single_obj"
  object_frame: "object_frame"
  scale: [0.1,0.1,0.1]
  position: [0,0,1]
  orientation: [0,0,0,1]


keypoints: [0,0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0]      # object pose, position + oreientation(quat)

approx_params:
  primitive_numbers: 1
  bottom_ind: 0